README.md

```diff
@@ -1,41 +1,40 @@
-# BOOKI: Exemple of a travel agency website.
-PROJECT 2 of OpenClassroom's Web developer Course
+# 😃 BOOKI: Exemple of a travel agency website.
+This website is my first achievement as a web developer👏. I did it during my boot camp at OpenClassroom's Web developer.
 
-## Use the full range of knowledge acquired during the courses.
+## 👀 Here you can check the final result
+https://645a7889605f5a058dad93d1--incredible-shortbread-990e7e.netlify.app/#
+
+## ⭐️ Use the full range of knowledge acquired during the courses.
 To realise this project there was some technical skill required, such as:
 
 
-* html
+* 🎯 `HTML`
   * Build the page using the right tabs
   * dividing the page in blocks from the wireframe to ensure a responsive design
   * Search engine optimization
   * Good syntaxe, comments and indent to make the code easily maintened and readable
   * Use css properties in order to not repeat the same code multiple time
```

Comment:
Should be in the CSS part ?

Reply:
i was talking about css class in the html code